sklearn/tree/tests/test_tree.py

```diff
@@ -1303,3 +1303,21 @@ def test_public_apply():
 
     for name in SPARSE_TREES:
         yield (check_public_apply_sparse, name)
+
+
+def check_presort_sparse(est, X, y):
+    assert_raises(ValueError, est.fit, X, y )
+
+def test_presort_sparse():
+    ests = (partial(DecisionTreeClassifier, presort=True)(), 
+            partial(DecisionTreeRegressor, presort=True)())
```

Comment:
I usually prefer to be explicit as opposed to implicit in the case of unit tests, so that it is clear what is being tested. What are your thoughts?